source/agora/consensus/data/UTXO.d

```diff
@@ -53,6 +53,133 @@ public struct UTXO
     }
 }
 
+/*******************************************************************************
+
+    This is the the base class of a UTXO storage.
+    It is currently inherited by UTXOSet and TestUTXOSet.
+
+*******************************************************************************/
+
+abstract class UTXOCache
```

Comment:
First thing, don't add code in one commit just to move it all in the next commit.
Second, instead of moving `UTXOCache` in the `data` package, we should move `TestUTXOSet` next to `UTXOCache`.